documentation/modules/ROOT/pages/integrations/cloudevents.adoc

```diff
@@ -133,10 +144,10 @@ It is also possible to use Avro for the envelope as well as the `data` attribute
 // Title: Example of configuring `CloudEventsConverter`
 == Example configuration
 
-Configure `io.debezium.converters.CloudEventsConverter` in your {prodname} connector configuration. Following is an example of configuring `CloudEventConverter` to emit change event records that:
+Configure `io.debezium.converters.CloudEventsConverter` in your {prodname} connector configuration. Following is an example of configuring `CloudEventsConverter` to emit change event records that:
 
 * Use JSON as the envelope
-* Contain Avro data that adheres to the schema at http://schema-registry:8081 
+* Use the schema registry at \http://schema-registry:8081 to serialize the `data` attribute as binary Avro data
```

Comment:
We use "my-registry" in other places in this doc. Let's converge on one example value everywhere. And is the added "\" before the URL intentional?

Reply:
I replaced `http://schema-registry:8081` with `http://my-registry/schemas/ids/1`.
The \ before the URL renders text rather than an active link so the reader does not see a clickable link. So yes, intentional.

Reply:
Gotcha. Nice trick with the backslash, didn't know that one :)